CONTRIBUTING.md

```diff
@@ -60,3 +60,15 @@ All pull requests need to have linting, unit tests and integration tests passing
 
 [Instructions for how to run tests are here in the Readme](https://github.com/bbc/simorgh/blob/latest/README.md#tests).
 
+### Run tests on your PR
```

Comment:
This isn't quite accurate and is definitely misleading. PRs against the simorgh repo will have tests run by our travisCI. These instructions are for how to have tests run on branches on the fork and PRs on the fork which are also to the fork (which is not default behaviour of a fork PR).